app/views/layouts/spina/journal/primer_theme/application.html.haml

```diff
@@ -0,0 +1,38 @@
+!!!
+%html{ lang: I18n.locale, data: { color_mode: 'auto', light_theme: 'light', dark_theme: 'dark' } }
+  %head
+    %meta{ charset: 'utf-8' }
+    %meta{ name: 'author', content: author }
+    %meta{ name: 'description', content: description }
+    %meta{ name: 'viewport', content: 'initial-scale=1.0' }
+    %meta{ name: 'og:title', content: t(:'.title', title: title, suffix: current_account.name) }
+    %meta{ name: 'og:description', content: description }
+    %meta{ name: 'turbolinks-cache-control', content: 'no-preview' }
+    %base{ target: '_blank' }
+    = csrf_meta_tags
+    %title= t(:'.title', title: seo_title, suffix: current_account.name)
+    = render 'spina/shared/google_site_verification'
+    = render 'spina/shared/analytics'
+    = stylesheet_link_tag 'spina/conferences/primer_theme/application', media: 'all', data: { turbolinks_track: true }
+    = javascript_include_tag 'spina/conferences/primer_theme/application', data: { turbolinks_track: true }
+    = javascript_include_tag 'turbo', type: 'module'
+    %script{ type: 'module', src: asset_path('@github/details-dialog-element/dist/index.js') }
+    %script{ type: 'module', src: asset_path('@github/include-fragment-element/dist/index.js') }
+    %script{ type: 'module', src: asset_path('@github/filter-input-element/dist/index.js') }
+    %script{ type: 'module', src: asset_path('@github/details-menu-element/dist/index.js') }
+    = yield :head
+  %body
+    - if current_spina_user.present?
+      .admin= render 'spina/shared/admin_bar'
+    %header= render 'navigation'
+    -# = render 'submissions_alert'
```

Comment:
✂️?

Reply:
Also, if this is the only difference from the source file, maybe we could modify the original to accommodate this?

Reply:
Yeah can do. I duplicated the file preemptively in case I needed to adjust anything but nothing arose. I remember why I didn't delete it though - in future I will have submissions details displayed, so a similar line might be needed in future. Combining the files into one works - but not of course if we split the gems.